### DISCUSSION TITLE
Test runner not launching

### DISCUSSION THREAD

**Discussion by prathap1990** [cypress-io/cypress#9226] [CI setup]
Hi All,

I am new to using cypress. I have created a sample test script. I am using visual studio code for maintaining the test scripts. I am trying to launch the test runner from console and command prompt. I am using "npx cypress open" command to launch the test runner. I can see the test runner open in the task bar but the window does not come up. Has anyone experienced the same issue? How to resolve this issue? 
Appreciate your help. Thanks!

**Comment by AishwaryaRavindra** [comment]
Can you give the Solution for this issue because i am facing same problem....
